HW-PHP-2022-12-04/task-1.php

```diff
@@ -0,0 +1,8 @@
+<?php
+//Разработайте функцию для вычисления суммы произвольного количества заданных элементов.
+$str = (string) readline ("Enter elements: ");
+$arr = explode(' ', $str);
+function sum ($arr) {
+    echo array_sum($arr);
+}
+sum($arr);
```

Comment:
тогда функцию так попробовать вызвать

```suggestion
sum(...$arr);
```

т.е. это будет ~ `sum(1, 2, 3, 4, ..., n);`